docs/user-guide/how-to-deploy.rst

```diff
@@ -1,21 +1,64 @@
-:desc: Run and ship your Rasa assistant with Docker containers on any
-       Docker-compatible machine or cluster.
+:desc: How to deploy your Rasa Assistant with Docker Compose or Kubernetes/Openshift
 
-.. _running-rasa-with-docker:
+.. _deploying-your-rasa-assistant:
 
-Running Rasa with Docker
-========================
+Deploying your Rasa Assistant
+===============================
 
 .. edit-link::
 
-This is a guide on how to build a Rasa assistant with Docker.
-If you haven't used Rasa before, we'd recommend that you start with the :ref:`rasa-tutorial`.
+This page explains when and how to deploy an assistant built with Rasa. 
+It will allow you to make your assistant available to users and set you up with a production-ready environment.
 
 .. contents::
    :local:
+   :depth: 2
+
+
+When to deploy your assistant
+--------------------------------
+
+.. raw:: html
+
+    The best time to deploy your assistant and make it available to test users is once it can handle the most important happy paths, or is what we call a <a style="text-decoration: none" href="https://rasa.com/docs/rasa/glossary">minimum viable assistant</a>.
```

Comment:
[](url)
```suggestion
    The best time to deploy your assistant and make it available to test users is once it can handle the most important happy paths or is what we call a <a style="text-decoration: none" href="https://rasa.com/docs/rasa/glossary">minimum viable assistant</a>.
```